llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1411,6 +1410,21 @@ PreservedAnalyses AddressSanitizerPass::run(Module &M,
       ClUseStackSafety ? &MAM.getResult<StackSafetyGlobalAnalysis>(M) : nullptr;
 
   if (Triple(M.getTargetTriple()).isSPIR()) {
```

Comment:
I just noticed, can you please change this check to
```
 if (Triple(M.getTargetTriple()).isSPIROrSPIRV()) {
```
We want to use the SPIR-V backend soon and that uses the SPIRV triple. Thanks.

Reply:
Sure

Reply:
https://github.com/intel/llvm/pull/14413